engine/src/main/java/org/terasology/monitoring/impl/PerformanceMonitorImpl.java

```diff
@@ -40,8 +41,10 @@
 public class PerformanceMonitorImpl implements PerformanceMonitorInternal {
     private static final int RETAINED_CYCLES = 60;
     private static final double DECAY_RATE = 0.98;
+    private static final Activity OFF_THREAD_ACTIVITY = new NullActivity();
```

Comment:
NullActivity is also used by the NullPerformanceMonitor, since it doesn't do anything. That doco should be in the PerformanceMonitorImpl on the offThreadActivity variable.